tests/rptest/services/redpanda.py

```diff
@@ -1009,7 +1009,7 @@ def prune(node):
                 f"sed -i -E -e '/TRACE|DEBUG/d' {RedpandaService.STDOUT_STDERR_CAPTURE} || true"
             )
 
-        self._for_nodes(self.nodes, prune, parallel=True)
+        self.for_nodes(self.nodes, prune, parallel=True)
```

Comment:
should `parallel` be removed from here now?

Reply:
Yes, fixed.